Lib/test/test_uuid.py

```diff
@@ -4,12 +4,17 @@
 import contextlib
 import io
 import os
+import pickle
```

Comment:
Why is this import added?

Reply:
this was to try and get merge to pass. I have now tried to add the current state of the test to my test (manual patch) as git rebase/merge of a single file is beyind my understanding of git.

Reply:
well, current status - since merge won't accept it, and the manual merge of the pickle addition does not work I'll probably have to start from scratch again (about the 4th time now iirc).

I'll get back to this as soon as I can - but it seems I have been "pickled" :p@me.